test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/io/OutputStreamWriterTest.scala

```diff
@@ -134,4 +134,9 @@ class OutputStreamWriterTest {
     testW({ osw => osw.write("ab\ud83d"); osw.close() },
         Array('a', 'b', '?'), alreadyFlushed = true)
   }
+
+  @Test def constructor_throw_UnsupportedEncodingException_if_unsupported_charset_name_given(): Unit = {
+    expectThrows(classOf[UnsupportedEncodingException],
+      new OutputStreamWriter(new ByteArrayOutputStream(), "UNSUPPOTED-CHARSET"))
```

Comment:
UNSUPPO**R**TED? Or intended?

Reply:
Fixed typo in https://github.com/scala-js/scala-js/pull/3687/commits/a1d0dce3e24e6580e8dfdd4b6854d446d47ed736